pkg/core/interop/crypto/interop.go

```diff
@@ -9,6 +9,7 @@ import (
 var (
 	ecdsaVerifyID        = emit.InteropNameToID([]byte("Neo.Crypto.ECDsaVerify"))
 	ecdsaCheckMultisigID = emit.InteropNameToID([]byte("Neo.Crypto.ECDsaCheckMultiSig"))
+	sha256ID             = emit.InteropNameToID([]byte("Neo.Crypto.Sha256"))
```

Comment:
Take a look at neo-project/neo#1419, it's post-preview2, but it's a compatible change wrt this hash function addition (secp256k1 is another topic, I'll open an issue for it, but that's post-0.90.0 for us). I think we can add it in this fashion (`SHA256` vs `Sha256`) right into `interops.go`.

Reply:
`core` is already to big, the point of moving some interops out was exactly ability to define them out of `core` (`crypto` in this case).

Reply:
I mean interop name to function binding, not function implementation.

Reply:
Oh, yeah, fixed.